src/main/java/duke/tasks/TaskList.java

```diff
@@ -0,0 +1,80 @@
+package duke.tasks;
+
+import duke.Duke;
+
+import java.util.ArrayList;
+
+/**
+ * Class that represents a list of tasks.
+ */
+public class TaskList {
+    private final ArrayList<Task> taskList;
+
+    /**
+     * Constructor to make a task list.
+     *
+     * @param list ArrayList of Tasks.
+     */
+    public TaskList(ArrayList<Task> list) {
+        taskList = list;
+    }
+
+    /**
+     * Find the number of Tasks.
+     *
+     * @return
+     */
+    public int length() {
+        return taskList.size();
+    }
```

Comment:
I think you should put what the method returns after the ```@return``` tag.
```suggestion
    /**
     * Find the number of Tasks.
     *
     * @return The number of tasks in the task list.
     */
    public int length() {
        return taskList.size();
    }
```